packages/camera/camera_android_camerax/lib/camera_android_camerax.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'camera_android_camerax_platform_interface.dart';
+
+/// Android Camera implented with the CameraX library.
+class CameraAndroidCamerax {
```

Comment:
This class shouldn't exist by default; the starting point for a federated implementation is just an implementation of the platform interface.